FreeRTOS/FreeRTOS
# Resolve stdio and stream_buffer link errors in RX64M demo (#42)

Resolve stdio and stream_buffer link errors in RX64M demo (#42)

Description
-----------
`.cproject`: Added stdio.h to standard library build. This is required because Common_Demo_Source\Minimal\MessageBufferDemo.c (and others) use `sprintf`.

`.project`: Included stream_buffer.c in the FreeRTOS source directory. This is required because Common_Demo_Source\Minimal\MessageBufferDemo.c uses numerous stream buffer functions such as `vStreamBufferDelete`.

Test Steps
-----------
See #42 for reproduction steps.
Tested by doing a git clone of this fork into a clean workspace, importing the project into Renesas E2 Studio IDE, and pressing "Build All". Verified that no errors occurred. Then loaded it onto the development board and verified that LED0 was blinky.

Related Issue
-----------
N/A

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
